src/spec/activity-service.ts

```diff
@@ -20,22 +20,72 @@ export class DecodeResult {
 
 export class EncodeResult {
   err: string | null = null
+  deserializeInputTook: number
   encodeTook: number
   serializationTook: number
   totalElapsed: number
   fileName: string
   fileType: string
-  fileBytes: Uint8Array
+  filesBytes: Uint8Array
```

Comment:
just wondering, is it bytes of multiple files?

Reply:
yes its array of bytes ([][]byte{}), but it looks like I haven't update the typescript side, nice catch 👍

Reply:
This has been updated too ya, I think this PR is complete, do you have any more concern?